docker-compose.yml

```diff
@@ -170,16 +170,17 @@ services:
       args:
         BASE_IMAGE: 'sentry-onpremise-local'
     command: '"0 0 * * * gosu sentry sentry cleanup --days $SENTRY_EVENT_RETENTION_DAYS"'
-  nginx:
+  frontend:
     << : *restart_policy
     ports:
       - '9000:80/tcp'
-    image: "nginx:1.16"
-    volumes:
-      - type: bind
-        read_only: true
-        source: ./nginx
-        target: /etc/nginx
+    image: 'us.gcr.io/sentryio/frontend:latest'
+    environment:
+      UPSTREAM_WEB: web:9000
```

Comment:
Technically this is a change in behavior. I'm not sure if you cared before, but if `web` got a new IP behind it, it'd be broken forever before.

We now force DNS resolution at runtime with `FORCE_DNS_RESOLVE`. If you want to remove that, feel free and just set it to `false`, and it'll mimic current behavior. At that point, could also drop `DNS_RESOLVER`.

Reply:
I think this is a good bugfix as we would never want to end up in a permanently broken state?